src/components/Game.tsx

```diff
@@ -0,0 +1,65 @@
+import { type } from "@testing-library/user-event/dist/type";
+import { useMemo, useState } from "react";
+import Board from "./Board";
+
+function Game() {
+  const resetbox: Array<string> = Array(9).fill("");
+  const [xIsNext, setxIsNext] = useState<boolean>(true);
+  const [box, setBox] = useState<Array<string>>(resetbox);
+
+  const handleClick = (index: number) => {
+    if (calculateWinner(box) || box[index]) {
+      return;
+    }
+    box[index] = xIsNext ? "X" : "O";
+    setBox(box);
+    setxIsNext(!xIsNext);
```

Comment:
Not sure let's wait for others to confirm 😿 
```suggestion
    setxIsNext((prevXIsNext) => !prevXIsNext);
```

Reply:
both ways are ok

Reply:
I see, Thanks! 🚀